pac2go.sh

```diff
@@ -1,25 +1,25 @@
-#!/usr/bin/env bash
+###############################################################################
+#
+# This script regenerates the source files that embed the pac-cmd executable.
+#
+###############################################################################
 
 function die() {
   echo $*
   exit 1
 }
 
-function gen() {
-  bin=$1
-  dst=$2
-  if [ -f "$bin" ]
-  then
-    echo Generating $dst from $bin...
-    2goarray pacBytes pac < $bin > $dst
-  else
-    echo $bin does not exist, skipping...
-  fi
-}
+if [ -z "$BNS_CERT" ] || [ -z "$BNS_CERT_PASS" ]
+then
+  die "$0: Please set BNS_CERT and BNS_CERT_PASS to the bns_cert.p12 signing key and the password for that key"
+fi
+
+BINPATH=../pac-cmd/binaries
+
+osslsigncode sign -pkcs12 "$BNS_CERT" -pass "$BNS_CERT_PASS" -in $BINPATH/windows/pac -out $BINPATH/windows/pac || die "Could not sign windows"
+codesign -s "Developer ID Application: Brave New Software Project, Inc" -f $BINPATH/darwin/pac || die "Could not sign macintosh"
 
-which 2goarray > /dev/null || go get github.com/cratonica/2goarray/...
-which 2goarray > /dev/null || die "Please install 2goarray manually, then try again"
-cmd_path=$GOPATH/src/github.com/getlantern/pac-cmd
-gen $cmd_path/pac pac_bytes_darwin.go
-gen $cmd_path/pac-linux pac_bytes_linux.go
-gen $cmd_path/pac.exe pac_bytes_windows.go
+go-bindata -nomemcopy -nocompress -pkg pac -prefix $BINPATH/darwin -o pac_bytes_darwin.go $BINPATH/darwin
```

Comment:
I switched to go-bindata because the compile-time performance is better.  Plus, it's consistent with what we're doing everywhere else.